api/app/models/concerns/spree/webhooks/has_webhooks.rb

```diff
@@ -0,0 +1,24 @@
+module Spree
+  module Webhooks
+    module HasWebhooks
+      extend ActiveSupport::Concern
+
+      included do
+        after_commit :execute_webhook_logic!,
+          on: %i[create destroy update],
+          unless: proc { self.class.module_parents.include?(Spree::Webhooks) }
+      end
+
+      private
+
+      def execute_webhook_logic!
+        uri = URI('https://google.com/')
```

Comment:
@sebastian-palma we should of course send this request in a background queue via ActiveJob ;)